### PR TITLE
created header and first block of main width adaptation bouth

### PR DESCRIPTION
**Pull request by vadosik15**
[DEMO LINK](https://vadosik15.github.io/layout_miami/)

### CODE REVIEW

File: src/styles/blocks/icons.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  &--menu {
+    width: 18px;
+    height: 12px;
+    background-image: url(../images/menu_icon.svg);
+    transition-duration: 0.3s;
+
+    @include onDesctop {
+      height: 18px;
+      width: 24px;
```
Comment (Lilu580) [inline src/styles/blocks/icons.scss:16]: on table you should use another image for burger

File: src/styles/mixins.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  @media (min-width:744px) {
+    @content;
+  }
+}
+
+@mixin onPhone {
+  @media (max-width:744px) {
+    @content;
+  }
+}
```
Comment (Lilu580) [inline src/styles/mixins.scss:17]: you can do this project without this mixin,just with method mobile-first

File: src/styles/emets/placeholder.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+%textContactNam {
+  color: $colorContact;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 140%;
+}
+%textLink {
+  font-weight: 400;
+  padding-top: 8px;
```
Comment (Lilu580) [inline src/styles/emets/placeholder.scss:9]: you have this paffing for elements of contact block,this elements should have margins bottom

**Review by Lilu580 — COMMENTED** [review]
Check cheklist 1.you have no favicon,3.All Logos on the page should be links to home page 6.Same when you click on logo. There shouldn't be any error.9.Location-related addresses / links should open google maps in a new tab target="_blank.14.Form shouldn't be submitted if some of the fields are not filled.15.Page shouldn't be reloaded on form submit.16Add a smooth scroll for the whole page.19Remember to reset default margins for headings (h1, h2, h3, etc.).

**Review by illiarerun — CHANGES_REQUESTED** [review]
- checklist 6th item
image

- checklist 5th item
image

- add transitions here
image

- and here
image

**Review by valeriiayarzhemska — COMMENTED** [review]
platform sent me here, but its closed

**Pull request closed without merging**